parquet/src/main/java/org/apache/iceberg/parquet/ParquetBloomRowGroupFilter.java

```diff
@@ -114,10 +119,17 @@ private boolean eval(
 
       Set<Integer> filterRefs =
           Binder.boundReferences(schema.asStruct(), ImmutableList.of(expr), caseSensitive);
-      // If the filter's column set doesn't overlap with any bloom filter columns, exit early with
-      // ROWS_MIGHT_MATCH
-      if (!filterRefs.isEmpty() && Sets.intersection(fieldsWithBloomFilter, filterRefs).isEmpty()) {
-        return ROWS_MIGHT_MATCH;
+      if (!filterRefs.isEmpty()) {
+        Sets.SetView overlappedBloomFilters = Sets.intersection(fieldsWithBloomFilter, filterRefs);
+        if (overlappedBloomFilters.isEmpty()) {
+          // If the filter's column set doesn't overlap with any bloom filter columns, exit early
+          // with ROWS_MIGHT_MATCH
+          return ROWS_MIGHT_MATCH;
+        } else {
+          LOG.debug(
```

Comment:
What about this? It seems descriptive but shorter to fit on a single line.

```
LOG.debug("Using Bloom filters for columns with IDs: {}", overlappingBloomFilterFieldIds);
```

Reply:
Changed. Thanks!